features/FxA-97-password-strength/README.md

```diff
@@ -0,0 +1,67 @@
+# Strengthening Passwords
+
+## Problem Summary
+
+[comment]: # (Talk about current goals of the system and how they aren't being met. What outcomes are we not delivering? This can also include an explicit request for improvement that doesn't dictate a specific or concrete solution.)
+
+Users possess varying levels of understanding of what they are signing up for, and what is at risk. When syncing Firefox data (namely passwords) to a cloud-based account, reusing a password for their account puts them at greater risk to online attacks.
+
+### Assumptions
+
+[comment]: # (This is where you talk about what you assume to be true. This could include assumptions around what users are doing, errors, gaps, etc., based on anecdotes, opinions, testing, or data.)
+
+Since account registration has appeared during First Run in an iframe, we are receiving are more users, but with less of an understanding of the significance of making an account. Since the Choose What To Sync screen has launched, we've noticed that First Run users are twice as likely to abandon during sign-up than users from other entry points.
+
+## Outcomes
+
+[comment]: # (What are the outcomes you want to achieve? What is the success criteria?)
+
+We feel that we could better serve user security and our [Data Privacy Principles](https://www.mozilla.org/en-US/privacy/principles/) by being more transparent about the risks and benefits of signing up for an account. This will hopefully result in stronger passwords as measured by our bloom filter, and also result in fewer account sign-ups (as many users will realize it's not something they need or want). Another goal is increasing multi-device usage, which in turn should result in an increase in overall Firefox engagement.
```

Comment:
The second goal (increasing multi-device usage) seems orthogonal to measuring password strength unless messaging about using Fx on other devices is also included.